reference/installation/config.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
this looks like a good config file for OS, but should we maintain a second config file for enterprise?

Reply:
Yeah but these are not the enterprise docs so there is no need to have that here.

Reply:
oh duh